app/src/main/res/values/strings.xml

```diff
@@ -145,12 +145,12 @@
     <!-- Search Fragment -->
     <!-- Button in the search view that lets a user search by scanning a QR code -->
     <string name="search_scan_button">Scan</string>
-    <!-- Button in the search view that lets a user search by using a shortcut -->
-    <string name="search_engines_shortcut_button">Search Engine</string>
+    <!-- Button in the search view that lets a user change their search engine -->
+    <string name="search_engines_button">Search engine</string>
     <!-- Button in the search view when shortcuts are displayed that takes a user to the search engine settings -->
     <string name="search_shortcuts_engine_settings">Search engine settings</string>
     <!-- Header displayed when selecting a shortcut search engine -->
-    <string name="search_engines_search_with">This time, search with:</string>
```

Comment:
This was renamed in #12830, original name was `search_shortcuts_search_with_2` which I moved down below and marked as `DEPRECATED`